platforms/quorum/configuration/roles/setup/geth/tasks/main.yaml

```diff
@@ -39,11 +31,20 @@
     state: directory
   when: geth_stat_result.stat.exists == False
 
-# This task puts the geth binary to above created bin directory
+# This task extract the geth tarball
+- name: Extract the Geth tarball
+  unarchive:
+    src: "{{ bin_install_dir }}/quorum/geth-alltools-linux-amd64-1.10.0-56dec25a.tar.gz"
+    dest: "{{ bin_install_dir }}/geth"
+    copy: no
+  become: yes
+  when: geth_stat_result.stat.exists == False
+
+# This task puts the geth binary to the bin directory created in the above task
 - name: Copy geth binary to destination directory
   copy:
-    src: "{{ bin_install_dir }}/quorum/build/bin/geth"
+    src: "{{ bin_install_dir }}/geth/geth-alltools-linux-amd64-1.10.0-56dec25a"
```

Comment:
This looks fine, why is the same logic not used for bootnode?

Reply:
done